Assets/_Project/Code/Scripts/Category draw/wheel.cs

```diff
@@ -0,0 +1,139 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+public class Wheel : MonoBehaviour
+{
+    float delta, angleStep, angleStepRad;
+
+    public bool spinning = false;
+    public float velocity = 0f;
+    public int numberOfSegments = 31;       // Ilo�� segment�w ko�a
+
+    public GameObject segmentPrefab;        // Prefab pojedynczego segmentu
+
+    public delegate void WheelStoppedHandler(int wynik);
+    public event WheelStoppedHandler OnWheelStopped;
```

Comment:
tutaj też, domyślnie pola powinny być prywatne, dodaj gettery jeżeli potrzebujesz odczytać wartości. Jeżeli chcesz widzieć zmienne w inspectorze unity, to [użyj tego](https://www.reddit.com/r/Unity3D/comments/dyn2xs/unity_tip_hide_public_variables_and_show_private/?rdt=57804)